.github/workflows/lint.yml

```diff
@@ -26,6 +26,6 @@ jobs:
       - name: Analyzing code with black
         run: python -m black --check --diff .
       - name: Analysing code with pylint
-        run: python -m pylint --rcfile=.github/workflows/.pylintrc $(git ls-files '*.py')
+        run: python -m pylint $(git ls-files '*.py')
```

Comment:
can you keep the `rcfile` flag, just redirect it? I understand that the location now corresponds to the default, but I'd still like to keep this explicit so that anyone looking through our CI will be pointed to the file.